src/main/java/edu/ucla/library/iiif/fester/verticles/V3ManifestVerticle.java

```diff
@@ -467,6 +471,16 @@ private void updatePages(final Message<JsonObject> aMessage) throws JsonProcessi
 
                 canvas.setDuration(duration).setThumbnails(new ImageContent(thumbnail));
                 canvas.paintWith(aMinter, audios);
+
+                // Possibly modify the annotation created with the above call to paintWith
+                CsvParser.getMetadata(columns, aCsvHeaders.getWaveformIndex()).ifPresent(waveformURI -> {
+                    final SeeAlso waveform = new SeeAlso(waveformURI, ResourceTypes.DATASET)
+                            .setProfile("http://waveform.prototyping.bbc.co.uk").setFormat(MediaType.OCTET_STREAM);
```

Comment:
Shouldn't the profile URI be a constant/config param, to make it easier to change (if needed) or other users to customize?